docs/production-deployment/self-hosted-guide/archival.mdx

```diff
@@ -27,13 +27,13 @@ This feature is helpful for compliance and debugging.
 
 Temporal's Archival feature is considered **experimental** and not subject to normal [versioning and support policy](/clusters#versions-and-support).
 
-Archival is not supported when running Temporal through Docker and is disabled by default when installing the system manually and when deploying through [helm charts](https://github.com/temporalio/helm-charts/blob/master/templates/server-configmap.yaml) (but can be enabled in the [config](https://github.com/temporalio/temporal/blob/master/config/development.yaml)).
+Archival is not supported when running Temporal through Docker and is disabled by default when installing the system manually and when deploying through [helm charts](https://github.com/temporalio/helm-charts/blob/main/charts/temporal/templates/server-configmap.yaml) (but can be enabled in the [config](https://github.com/temporalio/temporal/blob/main/config/development.yaml)).
```

Comment:
This reads a bit verbose too me. How about something like:

Archival is not supported when running Temporal through Docker. It is also disabled by default when installing the system manually and when deploying through [helm charts](https://github.com/temporalio/helm-charts/blob/main/charts/temporal/templates/server-configmap.yaml). However, it can be enabled in the [config](https://github.com/temporalio/temporal/blob/main/config/development.yaml).